tests/e2e/util.go

```diff
@@ -5570,3 +5570,34 @@ func waitForPvcToBeDeleted(ctx context.Context, client clientset.Interface, pvcN
 	framework.Logf("Status of pvc is: %v", pvc.Status.Phase)
 	return waitErr
 }
+
+/* This util method fetches pvc events list and matches the events list with the
+specified error message and returns true if expected error found */
+func getEventsListAndVerifyPvcError(client clientset.Interface, namespace string,
```

Comment:
you can use `waitForEvent` for checking the event message

Reply:
Combinned the new method and the olde one . Now , getEventsListAndVerifyPvcError checks both Event's Reason and  Message

Reply:
done